app/models/competition.rb

```diff
@@ -2658,6 +2665,10 @@ def disconnect_all_payment_integrations
     competition_payment_integrations.destroy_all
   end
 
+  def total_payment_amount
+    registrations.joins(:registration_payments).sum('registration_payments.amount_lowest_denomination')
```

Comment:
For better readability, you can add a `has_many :registration_payments, through: :registrations` association above, and then simply do `registration_payments.sum(:amount_lowest_denomination)` here.

Reply:
Changed